tests/e2e/utils/mock-requests.js

```diff
@@ -867,4 +867,17 @@ export default class MockRequests {
 			methods
 		);
 	}
+
+	/**
+	 * Fulfills a mock GET request for the price benchmarks suggestions.
+	 *
+	 * @param {Object} payload - The payload to be returned in the mock response.
+	 * @return {Promise<void>} Resolves when the mock request is fulfilled.
+	 */
+	async fulfillPriceBenchMarkSuggestions( payload = [] ) {
```

Comment:
@ankitrox There's already a similar function which does the same thing: `fulfillPriceBenchmarkSuggestions`

Reply:
Removed this function.